TabloidMVC/Repositories/PostRepository.cs

```diff
@@ -33,7 +34,7 @@ public List<Post> GetAllPublishedPosts()
                               LEFT JOIN Category c ON p.CategoryId = c.id
                               LEFT JOIN UserProfile u ON p.UserProfileId = u.id
                               LEFT JOIN UserType ut ON u.UserTypeId = ut.id
-                        WHERE IsApproved = 1 AND PublishDateTime < SYSDATETIME()";
+                        WHERE IsApproved = 0 AND PublishDateTime < SYSDATETIME()";
```

Comment:
Same thing is happening here, when I add posts to test and then try to view them on the Post tab, they don't show up because this query is looking for posts that are unapproved. Later on it will be important to only show posts that are approved, but rn it seems like everything is automatically set to 1/approved